Docs main placeholder pr concept (#1026)
nephio-project/porch: docs/content/en/docs/2_concepts/_index.md

      package revision is created with its workspace name given a value of "V2", it could very well end up with a Revision
      of `4` once published

* ***[Placeholder package revision]({{% relref "placeholder-package-revision" %}})***: A package revision that tracks

>   the content of a package's configured Git branch (typically `main`). It is created automatically when the first revision

Reviewer: ```suggestion
  the most recent content of a package in its repository. It is created automatically when the first revision
```
